src/api/serversStatus.ts

```diff
@@ -0,0 +1,33 @@
+import cp from 'child_process'
```

Comment:
could you add a simple integration test case for this? That sending a requset to `/api/serverStatus` to get back `{}`. This way, we at least test that this endpoint must be present and active.

Also, I like this filename as `serverStatus` better than the plural server from as it is now.

Reply:
Nvm about the simple integration test case, its already in the PR. Just need to change file name.